python/tvm/contrib/cc.py

```diff
@@ -7,7 +7,8 @@
 def create_shared(output,
```

Comment:
since there is not too much reuse, maybe it is simpler to put the logics in ndk.py and not call create_shared here